openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -20,12 +20,13 @@
 from django.utils.http import urlencode
 from django.utils.timezone import now
 from oauth2_provider import models as dot_models
+# from oauth2_provider.contrib.rest_framework.permissions import TokenHasReadWriteScope
 from rest_framework import status
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.test import APIClient, APIRequestFactory
 from rest_framework.views import APIView
-from rest_framework_oauth import permissions
-from rest_framework_oauth.compat import oauth2_provider, oauth2_provider_scope
+from rest_framework_oauth.permissions import TokenHasReadWriteScope
+import provider as oauth2_provider
```

Comment:
I'm not clear on this change.

Reply:
I think I might remove the changes to this file from this PR. They can be included in a later PR